k8s-deployment-manifest-templates/deployment-mode/daemonset/container-insights-monitoring/quickstart/cwagent-fluent-bit-quickstart.yaml

```diff
@@ -370,7 +370,7 @@ data:
         Name                tail
         Tag                 host.dmesg
         Path                /var/log/dmesg
-        Parser              syslog
```

Comment:
This change is suspicious

Reply:
Nvm its #108 . This is fine.

Reply:
@SaxyPandaBear  There are some others changes that are similar but not being updated (e.g [Parser syslog not changing](https://github.com/aws-samples/amazon-cloudwatch-container-insights/blob/main/k8s-deployment-manifest-templates/deployment-mode/daemonset/container-insights-monitoring/fluent-bit/fluent-bit-compatible.yaml#L191)).

Reply:
So we still need to update others change and confirm the behavior is consistent between the changes before delivering the newest container insight yaml version.